README.md

```diff
@@ -13,14 +13,17 @@ A curated list of awesome things and projects built by Algerian developpers.
 ## Open Data
 - [Algerian Administrative Division](https://github.com/mohsenuss91/AlgerianAdministrativeDivision) - Wilayas - Daïras - baladiyahs
 - [Algerial Logos](https://github.com/mohsenuss91/AlgerianLogos) - Some Algerian official vectorized (.SVG) Logos
+- [algeria-cities](https://github.com/othmanus/algeria-cities) - A complete list of Algeria provinces and municipalities in csv, xlsx, sql, php and json formats.
 
 ## Applications
 - [Alfanous](https://github.com/Alfanous-team/alfanous) - Alfanous is an Arabic search engine API provides the simple and advanced search in Quran
+- [DzLibs](https://github.com/01walid/dzlibs) - A community-driven Algerian index of reusable assets and libraries.
```

Comment:
I really like those projects particularly but I want to keep some organisation in the list and if you can just put a small commit to those projects (if you think they are worth adding). I have no intention of blocking any particular project, everything is accepted. And any change to the rules will be discussed with the contributors. Obviously, I want to keep "a small" control on the list at start. But actually, I think giving more control to contributors and moving the list to a GitHub Organisation (like GitHubAlgeria) for example will be the best thing to do. I just want to make sure the "values" behind the list are well understood.

While I'm more keen to say that we should gather our efforts on a single repository (and move to an org as I said, instead of it been attached to a particular person) in order to build a solid and unique community, I can't stop you from forking the project. You know you have all the rights to do so.